cvat/settings/base.py

```diff
@@ -92,6 +92,7 @@ def generate_ssh_keys():
     'cvat.apps.authentication',
     'cvat.apps.documentation',
     'cvat.apps.git',
+#    'cvat.apps.google',
```

Comment:
Don't need to put comment here. It is enough just say how to do that in README file.

Reply:
@nmanovic Done!